doc/source/examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.rst

```diff
@@ -0,0 +1,209 @@
+~~~~~~~~~~~~~~~~~~~
+
+In order to produce a varying Reynolds number, we vary the kinematic viscosity of the fluid. We define the Reynolds number as :math:`Re = \frac{UD}{\nu}` where :math:`U` and :math:`D` are fixed at 1. Therefore, 
+the kinematic viscosity is defined as :math:`nu = \frac{1}{Re}`. The following block defines the viscosity for a Reynolds number of 1.
+
+.. code-block:: text
+
+    subsection physical properties
+        subsection fluid 0
+            set kinematic viscosity = 1
+        end
+    end
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~
+
+With the Reynolds number ranging from 0.1 to 1, the simulation can easily be solved by using a steady-state solver. The mesh is sufficiently refined to capture the flow around the particle and therefore doesn't 
+require any mesh adaptation.
+
+.. code-block:: text
+
+    subsection simulation control
+        set method            = steady
+        set number mesh adapt = 2
+        set output path       = ./
+        set output name       = out
+        set output control    = iteration
+        set output frequency  = 1
+        set subdivision       = 1
+    end
+
+---------------
+Results
+---------------
+
+The simulation of this case results in the following solution for the velocity and pressure field.
+
+Velocity:
+ 
+.. image:: images/flow-around-particle-sharp-velocity.png
+    :alt: Velocity field around the particle
+    :align: center
+
+Pressure: 
+
+.. image:: images/flow-around-particle-sharp-pressure.png
+    :alt: Pressure field around the particle
+    :align: center
+
+The drag coefficient is calculated for each Reynolds number using the following formula:
+
+.. math::
+
+    C_d = \frac{F_d}{0.5 \rho U^2 D} = \frac{2f_x}{U^2 D}
+
+Where :math:`F_d` is the drag force, :math:`\rho` is the fluid density, :math:`U` is the velocity, and :math:`D` is the diameter of the particle. The drag force is calculated as the force applied 
+on the particle in the flow direction, in this case using :math:`f_x` from the output.
+
+The following graph shows the drag coefficient as a function of the Reynolds number. The results are compared to the litterature values for a perpendicular cylinder with an aspect ratio of 2.
+
+.. image:: images/flow-around-particle-sharp-pressure.png
+    :alt: Drag coefficient as a function of Reynolds number
+    :align: center
+
+----------------------------
+Possibilities for Extensions
+----------------------------
+
+- To further validate the simulation results, the drag coefficient could be compared to the litterature for a wider range of Reynolds numbers.
+- It would be interesting to investigate the switch from inline to perpendicular flow for the particle and the effect on the drag coefficient. This could be done by varying the orientation of the particle in flow.
+- It would be interesting to investigate the effect of the finite size of the particle on the drag coefficient. This could be done by varying the aspect ratio of the particle.
```

Comment:
```suggestion
- It would be interesting to investigate the switch from inline to perpendicular flow for the particle and the effect on the drag coefficient. This could be done by varying the orientation of the particle in flow.
- It would be interesting to investigate the effect of the aspect ratio of the particle on the drag coefficient. 
```